app/components/agenda/edit.js

```diff
@@ -1,11 +1,11 @@
-import Component from '@glimmer/component';
-import { tracked } from '@glimmer/tracking';
-import { action } from '@ember/object';
-
+import Component from "@glimmer/component";
+import { tracked } from "@glimmer/tracking";
+import { action } from "@ember/object";
 
 export default class AgendaEditComponent extends Component {
   @action
-  async toggleGeplandOpenbaar(){
-    this.args.agendapunt.geplandOpenbaar=!this.args.agendapunt.geplandOpenbaar;
+  async toggleGeplandOpenbaar() {
+    this.args.agendapunt.geplandOpenbaar = !this.args.agendapunt
```

Comment:
nitpicking, but I really liked this better on one line, perhaps increase minimum line width in your editor a bit?